templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -5,41 +5,46 @@
                     <span class="ohos-tag__inner">Location</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'PER' %}" class="ohos-tag ohos-tag--person">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'PER' %}" class="ohos-tag ohos-tag&#45;&#45;person">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=PER" class="ohos-tag ohos-tag--person">
                     <span class="ohos-tag__inner">Person</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'ORG' %}" class="ohos-tag ohos-tag--organisation">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'ORG' %}" class="ohos-tag ohos-tag&#45;&#45;organisation">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=ORG" class="ohos-tag ohos-tag--organisation">
                     <span class="ohos-tag__inner">Organisation</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'MISC' %}" class="ohos-tag ohos-tag--misc">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'MISC' %}" class="ohos-tag ohos-tag&#45;&#45;misc">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=MISC" class="ohos-tag ohos-tag--misc">
                     <span class="ohos-tag__inner">Miscellaneous</span>
                 </a>
             </li>
 
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'DATE' %}" class="ohos-tag ohos-tag--other">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'DATE' %}" class="ohos-tag ohos-tag&#45;&#45;other">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=DATE" class="ohos-tag ohos-tag--other">
                     <span class="ohos-tag__inner">Date</span>
                 </a>
             </li>
 
-            <li>
-                {# There may be a better way to achieve this without having to set a new param #}
-                <a href="?{% query_string_include 'chart_data_type' 'ALL' %}" class="ohos-tag ohos-tag--clear">
-                    <span class="ohos-tag__inner">Clear all</span>
-                </a>
-            </li>
+<!--            <li>-->
+<!--                {# There may be a better way to achieve this without having to set a new param #}-->
+<!--                <a href="?{% query_string_include 'chart_data_type' 'ALL' %}" class="ohos-tag ohos-tag&#45;&#45;clear">-->
+<!--                    <span class="ohos-tag__inner">Clear all</span>-->
+<!--                </a>-->
+<!--            </li>-->
```

Comment:
1. same as above on commented.
2. Why is the Clear all functionality removed?

Reply:
As we are not keeping the selected items as persistant, I have removed the 'Clear all' button.